<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1182</width>
    <height>718</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Source Code Pro</family>
    <pointsize>14</pointsize>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sniffer</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QTableWidget" name="tableWidget">
       <property name="font">
        <font>
         <family>Droid Sans Georgian</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #FF00FF; border-radius: 5px;
border-color: rgb(252, 233, 79);</string>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="gridStyle">
        <enum>Qt::DotLine</enum>
       </property>
       <attribute name="horizontalHeaderVisible">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string>No.</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Time</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Source</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Destination</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Protocol</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Length</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Info</string>
        </property>
       </column>
      </widget>
      <widget class="QTreeWidget" name="treeWidget">
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #FF00FF; border-radius: 5px;
border-color: rgb(78, 154, 6);</string>
       </property>
       <property name="animated">
        <bool>true</bool>
       </property>
       <attribute name="headerVisible">
        <bool>false</bool>
       </attribute>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QTextEdit" name="textEdit">
       <property name="minimumSize">
        <size>
         <width>300</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Source Code Pro</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">border: 1px solid #FF00FF; border-radius: 5px;</string>
       </property>
       <property name="locale">
        <locale language="English" country="UnitedStates"/>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1182</width>
     <height>42</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Serif Tamil Slanted</family>
    </font>
   </property>
   <widget class="QMenu" name="menufile">
    <property name="title">
     <string>file</string>
    </property>
    <addaction name="actionopen"/>
    <addaction name="actionsave"/>
    <addaction name="separator"/>
    <addaction name="actionquit"/>
   </widget>
   <widget class="QMenu" name="menudevice">
    <property name="title">
     <string>device</string>
    </property>
    <addaction name="actionchange"/>
    <addaction name="actiondetail"/>
   </widget>
   <widget class="QMenu" name="menufilter">
    <property name="title">
     <string>filter</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionARP"/>
    <addaction name="actionRARP"/>
    <addaction name="separator"/>
    <addaction name="actionIP"/>
    <addaction name="separator"/>
    <addaction name="actionTCP"/>
    <addaction name="actionUDP"/>
    <addaction name="separator"/>
    <addaction name="actionICMP"/>
   </widget>
   <widget class="QMenu" name="menupacket">
    <property name="title">
     <string>packet</string>
    </property>
    <addaction name="actionstart"/>
   </widget>
   <addaction name="menufile"/>
   <addaction name="menudevice"/>
   <addaction name="menufilter"/>
   <addaction name="menupacket"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px solid #FF00FF; border-radius: 5px;
border-color: rgb(193, 125, 17);</string>
   </property>
   <property name="sizeGripEnabled">
    <bool>false</bool>
   </property>
  </widget>
  <action name="actionethernet">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ethernet</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionARP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ARP</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionRARP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>RARP</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionIP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>IP</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionTCP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>TCP</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionUDP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>UDP</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionICMP">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ICMP</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionopen">
   <property name="text">
    <string>open</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionsave">
   <property name="text">
    <string>save</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionquit">
   <property name="text">
    <string>quit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionchange">
   <property name="text">
    <string>change</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actiondetail">
   <property name="text">
    <string>detail</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
  <action name="actionstart">
   <property name="text">
    <string>start</string>
   </property>
   <property name="font">
    <font>
     <pointsize>13</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionquit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>540</x>
     <y>335</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
